app/webpacker/src/stylesheets/_help.scss

```diff
@@ -0,0 +1,24 @@
+.govspeak .attachment,
+.gem-c-govspeak .attachment {
+  margin-top: 30px;
```

Comment:
Can these be converted to using govuk responsive features - see

https://design-system.service.gov.uk/styles/spacing/#spacing-on-custom-components